geopyspark/geotrellis/key_conversion.py

```diff
@@ -0,0 +1,157 @@
+'''
+A module to provide facilites for converting between layout keys and spatial objects.  These facilities aim to bridge between the various layouts, ``SpatialKey``s, and geometry.
+'''
+
+from math import ceil
+
+from . import Extent
+import geopyspark as gps
+
+__all__ = ["KeyTransform"]
+
+class KeyTransform(object):
+    """Provides functions to move from keys to geometry and vice-versa.
+
+    Tile Layers have an underlying RDD which is keyed by either :class:`geopyspark.geotrellis.SpatialKey` or
```

Comment:
When referencing another object in the docstrings, a `~` needs to come before the path to the object (ie ``:class:`~geopyspark.geotrellis.SpatialKey` ``). Otherwise, I don't think it'll link correctly.